### PR TITLE
Layer norm fusion deepspeed stage3 changes

### PR DESCRIPTION
**Pull request by ajindal1**
### Description

Layer norm fusion changes required for deepspeed stage 3, also includes test case.

### Motivation and Context

It helps fusing layer norm for Deepspeed Stage 3. Added a test case scenario which ensures that the fusion is working properly for the scenario.

### CODE REVIEW

File: onnxruntime/test/testdata/transform/fusion/layer_norm_fusion_scale_bias.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import onnx
```
Comment (pengwa) [inline onnxruntime/test/testdata/transform/fusion/layer_norm_fusion_scale_bias.py:1]: add following in the beggining

```
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.
```

File: onnxruntime/test/optimizer/graph_transform_test_layernorm.cc [hunk 2]
```diff
@@ -429,6 +429,38 @@ TEST_F(GraphTransformationTests, SimplifiedLayerNormFusionTest) {
   }
 }
 
+TEST_F(GraphTransformationTests, LayerNormScaleBiasTest) {
+  constexpr const ORTCHAR_T* model_uri = MODEL_FOLDER "fusion/layer_norm_fusion_scale_bias.onnx";
```
Comment (pengwa) [inline onnxruntime/test/optimizer/graph_transform_test_layernorm.cc:433]: shall we give more details explainationg what the test is testing?

Comment (pengwa) [reply onnxruntime/test/optimizer/graph_transform_test_layernorm.cc:433]: for example: test the case when scale or bias are not graph input or graph initilizer.

**Review by pengwa — APPROVED** [review]

**Review by hanbitmyths — APPROVED** [review]
LGTM

**Pull request merged**